src/styles/blocks/mains.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 25px $calendarShihtPading 10px ;
+  gap: $gapForDays;
+  margin: auto;
+
+  width: $calendarMaxWidth;
+
+  @each $day in $days {
+    &--start-day-#{$day} > .calendar__day:first-child {
+      margin-left: $startDayShift;
+    }
+  }
+
+  @for $lastDay from 28 to 32 {
+    &--month-lenght-#{$lastDay} .calendar__day:nth-last-child(-n + 2) {
+      display: none;
+    }
+  }
```

Comment:
@for $i from 28 through 31 {
    &--month-length--#{$i}>&__day:nth-child(n + #{$i + 1}) {
      display: none;
    }
  }